### PR TITLE
add install info on desktop apps

### PR DESCRIPTION
**Pull request by youngcw**

### CODE REVIEW

File: docs/install/index.md [hunk 1]
```diff
@@ -1,13 +1,39 @@
...
+| Import transactions from files | X | |
+| All budgeting features (budgets, reports, schedules, etc) | X | |
+| Import or Export budget files | X | |
+| Use Actual on a mobile device | | X[^1] |
+| Use Actual in a web browser | | X[^1] |
+| Sync budget between devices| | X |
+| Use bank syncing (GoCardless or SimpleFIN)| | X |
+| Use the Actual API | | X |
+
+The standard way of using Actual is to setup a personal server and use a web browser for the application.  For quick testing or getting to know Actual before setting up a server, the [demo](https://demo.actualbudget.org) or the [desktop applications](https://github.com/actualbudget/actual/releases) are a good place to start.
```
Comment (psybers) [inline docs/install/index.md:16]: When used as a verb, it is "set up" and not "setup".
```suggestion
The standard way of using Actual is to set up a personal server and use a web browser for the application.  For quick testing or getting to know Actual before setting up a server, the [demo](https://demo.actualbudget.org) or a [desktop application](https://github.com/actualbudget/actual/releases) are a good place to start.
```

File: docs/install/index.md [hunk 2]
```diff
@@ -1,13 +1,39 @@
...
+
+## Server Optional Client Options
```
Comment (psybers) [inline docs/install/index.md:18]:
```suggestion
## Server-Optional Client Options
```

File: docs/install/index.md [hunk 3]
```diff
@@ -1,13 +1,39 @@
...
+
+Desktop applications are available for Windows, Mac, and Linux. These can be [downloaded from GitHub](https://github.com/actualbudget/actual/releases).  The desktop apps have the following benefits:
+* Can connect to a server (this gives them the ability to use the server based features)
+* Automated backups
+* Offline use is ready out of the box 
+
+## Server Based Client Options
```
Comment (psybers) [inline docs/install/index.md:25]:
```suggestion
## Server-Based Client Options
```

File: docs/install/index.md [hunk 4]
```diff
@@ -1,13 +1,39 @@
...
+
+The server provides a web based version of Actual.  This web app can be used in a browser as a standard web page to use view and edit your budget.  The web page can also be installed to your device.  For mobile devices an installed web page will work offline.
```
Comment (psybers) [inline docs/install/index.md:27]:
```suggestion
The server provides a web-based version of Actual.  This web app can be used in a browser as a standard web page to view and edit your budget.  The web page can also be installed on your device.  For mobile devices, an installed web page will work offline.
```

File: docs/install/index.md [hunk 5]
```diff
@@ -1,13 +1,39 @@
...
+| Use the Actual API | | X |
+
+The standard way of using Actual is to setup a personal server and use a web browser for the application.  For quick testing or getting to know Actual before setting up a server, the [demo](https://demo.actualbudget.org) or the [desktop applications](https://github.com/actualbudget/actual/releases) are a good place to start.
+
+## Server Optional Client Options
+
+Desktop applications are available for Windows, Mac, and Linux. These can be [downloaded from GitHub](https://github.com/actualbudget/actual/releases).  The desktop apps have the following benefits:
+* Can connect to a server (this gives them the ability to use the server based features)
+* Automated backups
+* Offline use is ready out of the box
```
Comment (psybers) [inline docs/install/index.md:23]:
```suggestion
Desktop applications are available for Windows, Mac, and Linux. These can be [downloaded from GitHub](https://github.com/actualbudget/actual/releases).  The desktop apps have the following benefits:
* Can connect to a server (this gives them the ability to use the server-based features)
* Automated backups
* Offline use is ready out of the box 
```

File: docs/install/index.md [hunk 6]
```diff
@@ -26,3 +52,6 @@ Content contained on external links is not managed or maintained by the Actual B
 
 - [Synology NAS](https://mariushosting.com/how-to-install-actual-on-your-synology-nas/)
 - [Home Assistant](https://github.com/sztupy/hassio-actualbudget/blob/main/README.md)
+
+[^1]: You technically don't need a server instance for this. You need to run either a web based client or a server, but a server is the same effort.
+[^2]: A portion of the cost to host on PikaPods is donated to the Actual Budget Project.  With that said, PikaPods is a very simple, and cost effective way, to host your server.
```
Comment (psybers) [inline docs/install/index.md:57]:
```suggestion
[^1]: You technically don't need a server instance for this. You need to run either a web-based client or a server, but a server is the same effort.
[^2]: A portion of the cost to host on PikaPods is donated to the Actual Budget Project.  With that said, PikaPods is a very simple, and cost-effective way, to host your server.
```

**Review by psybers — CHANGES_REQUESTED** [review]

**Review by psybers — APPROVED** [review]
LGTM.

**Review by carkom — APPROVED** [review]
Looks good!

**Pull request merged**